Sentinel commands have migrated in redis main

Redis pre release test throwing error in CI


- Test Python 3.9, Django 3.2, Redis.py master
- Test Python 3.9, Django 3.2 main, Redis.py master

```python
  self = Sentinel<sentinels=[127.0.0.1:26379]>, service_name = 'default_service'
  
      def discover_master(self, service_name):
          """
          Asks sentinel servers for the Redis master's address corresponding
          to the service labeled ``service_name``.
      
          Returns a pair (address, port) or raises MasterNotFoundError if no
          master is found.
          """
          for sentinel_no, sentinel in enumerate(self.sentinels):
              try:
  >               masters = sentinel.sentinel_masters()
  E               AttributeError: 'Redis' object has no attribute 'sentinel_masters'
  
  .tox/py39-dj32-redismaster/lib/python3.9/site-packages/redis/sentinel.py:211: AttributeError
  _____ ERROR at teardown of DjangoRedisCacheTestEscapePrefix.test_iter_keys _____
  
  self = <tests.test_backend.DjangoRedisCacheTestEscapePrefix testMethod=test_iter_keys>
  
      def tearDown(self):

```

--- a/changelog.d/542.bugfix
+++ b/changelog.d/542.bugfix
@@ -1,0 +1,1 @@
+Block use with broken redis-py 4.0.0 and 4.0.1

--- a/changelog.d/576.misc
+++ b/changelog.d/576.misc
@@ -1,0 +1,1 @@
+Unblock redis-py >=4.0.2

--- a/setup.cfg
+++ b/setup.cfg
@@ -38,10 +38,10 @@ packages =
     django_redis.compressors
 install_requires =
     Django>=2.2
-    redis>=3,<4
+    redis>=3,!=4.0.0,!=4.0.1
 
 [options.extras_require]
-hiredis = redis[hiredis]>=3,<4
+hiredis = redis[hiredis]>=3,!=4.0.0,!=4.0.1
 
 [coverage:run]
 omit =

